supertokens_python/recipe/emailpassword/syncio/__init__.py

```diff
@@ -42,8 +42,8 @@ def is_email_verified(user_id: str, user_context: Union[None, Dict[str, Any]] =
     return sync(is_email_verified(user_id, user_context))
 
 
-async def update_email_or_password(user_id: str, email: Union[str, None] = None,
```

Comment:
This fix should go in another PR to the main x.y branch as it is unrelated to this PR.

Reply:
You need to make another PR fixing this (to the latest X.Y branch).

Reply:
Created new PR https://github.com/supertokens/supertokens-python/pull/168/files